emulator/blockchain.go

```diff
@@ -1180,7 +1183,10 @@ func (b *Blockchain) executeNextTransaction(ctx fvm.Context) (*types.Transaction
 
 	b.currentCode = string(txnBody.Script)
 	b.currentScriptID = txnId.String()
-	if b.activeDebuggingSession {
+
+	pragmas := ExtractPragmas(b.currentCode)
+
+	if b.activeDebuggingSession && pragmas.HasPragma("debug") {
```

Comment:
can you move these magic strings to consts in the pragma file? That will also create a convenient place to document them all.